site/content/about/_index.md

```diff
@@ -26,39 +26,48 @@ We create partnerships with other organizations and communities; both those that
 
 ## Team
 
-Techqueria is possible thanks to the work of lots of people, organizers, volunteers, mentors, sponsors, and advisors; and we're thankful for all of them.
+Techqueria is an all-volunteer organiztion that is made possible thanks to the work of lots of organizers, mentors, sponsors, and advisors. Thank you all!
 
 ### Co-Founders
 
-- [David Silva](https://linkedin.com/in/dvidsilva), co-founder of Techqueria & senior developer at Slingshot Health
-- [Shashi Jain](https://www.linkedin.com/in/quahada), co-founder of Techqueria & software engineer at Diamond Web Services
+- [David Silva](https://linkedin.com/in/dvidsilva), Co-founder of Techqueria & Senior Developer at Slingshot Health
+- [Shashi Jain](https://www.linkedin.com/in/quahada), Co-founder of Techqueria & Software Engineer
 
 ### Organizers
 
 #### Bay Area
-
-- [Shashi Jain](https://www.linkedin.com/in/quahada), co-founder of Techqueria & software engineer at Diamond Web Services
-- [David Hincapie](https://www.linkedin.com/in/davidohincapie/), product specialist at Oculus VR
-- [Philip Ventura](https://www.linkedin.com/in/fvntr/), software engineer at Lending Club
+- [Rubi Sanchez](https://www.linkedin.com/in/rubisanchez), Product Manager at AdRoll
+- [Dalia Icedo](https://www.linkedin.com/in/daliaicedo), Freelance Product Designer
+- [Shashi Jain](https://www.linkedin.com/in/quahada), Co-founder of Techqueria & Software Engineer
+- [Kevin Hernandez](https://www.linkedin.com/in/kevdez) UI Developer at WalmartLabs
+- [Phillip Ventura](https://www.linkedin.com/in/fvntr/), Software Engineer
 
 #### Los Angeles
 
-- [Esteban Uribe](https://www.linkedin.com/in/estebanuribe/), co-founder of Techqueria's LA chapter & iOS Engineer at Tinder
-- [Hector Carmona](https://www.linkedin.com/in/hectoroddincarmona/), software engineer at Google
-- [Josue Bustos](https://www.linkedin.com/in/josuebustos), technical evangelist at Samsung
+- [Esteban Uribe](https://www.linkedin.com/in/estebanuribe/), Co-founder of Techqueria's LA chapter & iOS Engineer at Tinder
+- [Hector Carmona](https://www.linkedin.com/in/hectoroddincarmona/), Software Engineer at Google
+- [Josue Bustos](https://www.linkedin.com/in/josuebustos), Technical Evangelist at Samsung
 
 #### New York
 
-- [Tatiana Carett](https://www.linkedin.com/in/tatianacarett/), associate at Booz Allen Hamilton
-- [David Silva](https://linkedin.com/in/dvidsilva), co-founder of Techqueria & senior developer at Slingshot Health
+- [Tatiana Carett](https://www.linkedin.com/in/tatianacarett/), Associate at Booz Allen Hamilton
+- [David Silva](https://linkedin.com/in/dvidsilva), Co-founder of Techqueria & senior developer at Slingshot Health
+
+### Austin
+- [Andrea Flores](https://www.linkedin.com/in/andreaf), PhD Student at The University of Texas at Austin School of Information
+
+### Past Organizers
```

Comment:
Past organizers should be underneath developers header so the developers are current